src/db/models/users.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { sequelize } = require('../main');
+const { DataTypes } = require('sequelize');
+
+const Users = sequelize.define('User', {
```

Comment:
Consider using a consistent naming convention for models. Rename `Users` to `User` since the model represents a single user.